Members/DaytL.md

```diff
@@ -0,0 +1,15 @@
+## About
+Hello, I am Rishabh Raghuvanshi, currently pursuing B.Tech in Information Technology.
+
+## Place
+Indore, M.P
+
+## Programming Languages: 
+ -C++
```

Comment:
@DaytL  the syntax for bullet points should be:
`- C++`

Reply:
I have updated the Syntax.

Reply:
Thanks for feedback. I have resolved the issue.

Reply:
great